static/css/bootstrap/tests/css-tests.html

```diff
@@ -1360,8 +1360,8 @@ <h4>Error</h4>
     <footer class="footer">
       <div class="container">
         <p class="pull-right"><a href="#">Back to top</a></p>
-        <p>Designed and built with all the love in the world <a href="http://twitter.com/twitter" target="_blank">@twitter</a> by <a href="http://twitter.com/mdo" target="_blank">@mdo</a> and <a href="http://twitter.com/fat" target="_blank">@fat</a>.</p>
-        <p>Code licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0" target="_blank">Apache License v2.0</a>. Documentation licensed under <a href="http://creativecommons.org/licenses/by/3.0/">CC BY 3.0</a>.</p>
+        <p>Designed and built with all the love in the world <a href="http://twitter.com/twitter" target="_blank" rel="noopener noreferrer">@twitter</a> by <a href="http://twitter.com/mdo" target="_blank" rel="noopener noreferrer">@mdo</a> and <a href="http://twitter.com/fat" target="_blank" rel="noopener noreferrer">@fat</a>.</p>
+        <p>Code licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0" target="_blank" rel="noopener noreferrer">Apache License v2.0</a>. Documentation licensed under <a href="http://creativecommons.org/licenses/by/3.0/">CC BY 3.0</a>.</p>
```

Comment:
Let's not modify bootstrap code. I'm not even quite sure why we have committed this instead of using the version in `node_modules`. Revert this change, then either fix it so bootstrap is included from the node module, _or_ add an exception for bootstrap to your rule and raise an issue to fix the include.